app/models/job.rb

```diff
@@ -63,10 +63,14 @@ def check_active_on_destroy
     true
   end
 
-  def self.agent_state_update_queue(jobid, state, message = nil)
+  def self.agent_state_update_queue(jobid, _state, message = nil)
```

Comment:
@gtanzillo There is no caller. I was thinking that there could be some messages for this method in the queue during upgrade (which not very likely). If you are OK, i will remove it